runtimes/aleph-debian-12-python/create_disk_image.sh

```diff
@@ -37,7 +37,7 @@ locale-gen en_US.UTF-8
 echo "Pip installing aleph-sdk-python"
 mkdir -p /opt/aleph/libs
 # Fixing this protobuf dependency version to avoid getting CI errors as version 5.29.0 have this compilation issue.
-pip3 install --target /opt/aleph/libs 'aleph-sdk-python==2.0.0' 'aleph-message~=1.0.1' 'fastapi~=0.109.2' 'protobuf==5.28.3'
+pip3 install --target /opt/aleph/libs 'aleph-sdk-python==2.0.5' 'aleph-message~=1.0.5' 'fastapi~=0.120.1' 'protobuf==6.33.0'
```

Comment:
Remind me: is the CI rebuilding the diagnostic VM image here? Or where are you specifying the runtime to use for automated tests?